.github/workflows/lint_and_format.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         python -m pip install pyink==24.3.0
         pip install pytype
         pip install pylint
+        pip install .
```

Comment:
Hmm. But why didn’t we catch this error in the original PR?

Reply:
I also can see linter errors on main branch.

Reply:
Not sure, probably I tried it on file not importing urllib

Reply:
What kind of errors? Aim of this change and previous PR is to add linter checks for new code. For existing code type hints will be corrected in separate PR

Reply:
I mean after push changes github action didn't show us this error.

Reply:
Probably we need to revert `push`. https://github.com/AI-Hypercomputer/xpk/pull/227/files#diff-9be398388c4a96355ba1dcbf7c41cee8567621b5e50c1693da04a5639394ce42L17

Reply:
yep, aim of this pr is to remove error comming from missing urllib dependency

Reply:
I see, probably we need to fix it, before we have each check run twice(first time with pull request trigger and second time with push trigger)

Reply:
Also, we need to check the trigger for merges to the main branch.